include/gridtools/common/demangle.hpp

```diff
@@ -0,0 +1,118 @@
+// core::demangle
+//
+// Copyright 2014 Peter Dimov
+// Copyright 2014 Andrey Semashev
+//
+// Distributed under the Boost Software License, Version 1.0.
+// See accompanying file LICENSE_1_0.txt or copy at
+// http://www.boost.org/LICENSE_1_0.txt
```

Comment:
I think this is ok, but want to check what you say since I know you've had extensive licensing discussions around gridtools.

Reply:
Yes, I agree that should be fine.